client/state/selectors/can-current-user.js

```diff
@@ -17,12 +11,11 @@ import { isValidCapability } from 'calypso/state/current-user/selectors';
  * @param  {string}   capability Capability label
  * @returns {boolean}            Whether current user has capability
  */
-export const canCurrentUser = ( state, siteId, capability ) => {
-	if ( ! isValidCapability( state, siteId, capability ) ) {
-		return false;
+export default function canCurrentUser( state, siteId, capability ) {
```

Comment:
Removing the duplicate export is definitely good, but I would rather see us keep the named export, instead of the default one. Still, that's a matter of personal preference while we don't have any guidelines 👍